sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcDataRecord.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.jdbc;
+
+import java.io.Serializable;
+
+/**
+ * Container for JDBC data, providing the data details (column name, table name, payload, data
+ * type).
+ */
+public class JdbcDataRecord implements Serializable {
+
+  private String[] tableNames;
```

Comment:
If the user query is like this: 

```
select t1.id,t2.name from table1 t1, table t2 where t1.id = t2.id
```

each `JdbcDataRecord` can have different table.